File: dotnetcore/core/WebCore/Context/WebContextRepository.cs
namespace WebCore.Context
{
    using Platform.Context;
    using System;

    public class WebContextRepository : BaseContextRepository<WebContext>
    {
        private static readonly Lazy<WebContextRepository> _instance = new Lazy<WebContextRepository>(() => new WebContextRepository());

        public static WebContextRepository Instance
        {
            get
            {
                return _instance.Value;
            }
        }
    }
}

File: java/core/web/src/main/java/com/wiiee/core/web/context/WebContextRepository.java
package com.wiiee.core.web.context;

import com.wiiee.core.platform.context.BaseContextRepository;
import org.springframework.stereotype.Repository;

/**
 * Created by wang.na on 2017/03/20.
 */
@Repository
public class WebContextRepository extends BaseContextRepository {
    public WebContextRepository() {
        super("WebContextRepository");
    }
}
